helm/alfresco-content-services/requirements.yaml

```diff
@@ -10,8 +10,8 @@ dependencies:
   repository: https://kubernetes-charts.storage.googleapis.com/
   condition: postgresql.enabled
 - name: alfresco-search
-  version: 0.0.3
-  repository: http://kubernetes-charts.alfresco.com/stable
+  version: 0.0.4
+  repository: http://kubernetes-charts.alfresco.com/incubator
```

Comment:
We will need to keep a close eye on how we use incubator vs having stable EA candidates.  This is something we will have to figure out over time, so just worth keeping an eye on.